ntbs-integration-tests/NotificationPages/DeletePageTests.cs

```diff
@@ -0,0 +1,132 @@
+{
+    public class DeletePageTests : TestRunnerBase
+    {
+        protected override string PageRoute => Routes.Delete;
+
+        public DeletePageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory) { }
+
+        public static IList<Notification> GetSeedingNotifications()
+        {
+            return new List<Notification>()
+            {
+                new Notification(){ NotificationId = Utilities.DELETE_WITH_DESCRIPTION, NotificationStatus = NotificationStatus.Draft },
+                new Notification(){ NotificationId = Utilities.DELETE_NO_DESCRIPTION, NotificationStatus = NotificationStatus.Draft },
+            };
+        }
+
+        public static IEnumerable<object[]> DeleteRoutes()
+        {
+            yield return new object[] { Utilities.DRAFT_ID, HttpStatusCode.OK };
+            yield return new object[] { Utilities.DENOTIFIED_ID, HttpStatusCode.Redirect };
+            yield return new object[] { Utilities.NOTIFIED_ID, HttpStatusCode.Redirect };
+            yield return new object[] { Utilities.NEW_ID, HttpStatusCode.NotFound };
+        }
+
+        [Theory, MemberData(nameof(DeleteRoutes))]
+        public async Task GetDeletePage_ReturnsCorrectStatusCode_DependentOnId(int id, HttpStatusCode code)
+        {
+            // Act
+            var response = await client.GetAsync(GetPageRouteForId(id));
+
+            // Assert
+            Assert.Equal(code, response.StatusCode);
+
+            if (response.StatusCode == HttpStatusCode.Redirect)
+            {
+                Assert.Equal(BuildRoute(Routes.Overview, id), GetRedirectLocation(response));
+            }
+        }
+
+        [Fact]
+        public async Task Post_DeletesAndDisplaysConfirmationPage_IfModelValidWithNoDescription()
+        {
+            // Arrange
+            const int id = Utilities.DELETE_NO_DESCRIPTION;
+            var initialPage = await client.GetAsync(GetPageRouteForId(id));
+            var initialDocument = await GetDocumentAsync(initialPage);
+
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = id.ToString()
+            };
+
+            // Act
+            var result = await SendPostFormWithData(initialDocument, formData, "Confirm");
+
+            
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, result.StatusCode);
+
+            var postFormSubmissionResult = await GetDocumentAsync(result);
+            Assert.NotNull(postFormSubmissionResult?.QuerySelector(".return-to-homepage-link"));
+        }
+
+        [Fact]
+        public async Task Post_DeletesAndDisplaysConfirmationPage_IfModelValidWithDescription()
+        {
+            // Arrange
+            const int id = Utilities.DELETE_WITH_DESCRIPTION;
+            var initialPage = await client.GetAsync(GetPageRouteForId(id));
+            var initialDocument = await GetDocumentAsync(initialPage);
+
+            
+            const string reason = "A great reason";

```

Comment:
What's the need for it not inlined - were you planning on asserting based on it?

Reply:
Ah nope, was just following a pattern from tests that were - will inline